src/js/page_sections/Navmenu.js

```diff
@@ -84,17 +51,19 @@ class Navmenu extends Component {
 
     return parentLinks.length && (
       <div className="usa-grid-full">
-        <nav className={this.menuClassName()}>
+        <nav className={`coa-Navmenu ${this.props.isOpen ? 'coa-Navmenu--open' : ''}`}>
```

Comment:
Seems like you prefer this inline logic rather than calling a function that returns the proper string. Can you explain why this wasn't working well for you? For me, the code reads easier when the markup is clean and the classname comes back from a function.

Reply:
Totes true, with it being cleaner in a method.
This was phase one, before I import classnames, so all this class stuff can be inline and even cleaner -- without ternaries inline.